Makefile

```diff
@@ -74,6 +74,11 @@ test: build
 test-coverage:
 	${DOCKER_COMPOSE} run ${RESOURCES_CONTAINER} py.test --cov-report html --cov=app/ tests/
 
+# Usage: make test-single tests/unit/test_api_key.py::test_get_api_key
+.PHONY: test-single
+test-single: build
```

Comment:
This has nothing to do with this PR in particular, I just realized it was useful and wanted to use it